### DISCUSSION TITLE
Accessibility and Security Documentation for Hubs Cloud on AWS

### DISCUSSION THREAD

**Discussion by AbigailRose333** [mozilla/hubs#4787] [Hubs Cloud Questions] [answered]
Hi everyone!   We're trying to order Mozilla Hubs Cloud Enterprise on AWS through our organization, and we are required to ask vendors about security and accessibility information.  Basically, we just need to know if there is "VPAT" for Mozilla Hubs Cloud which outlines Section 508 compliance for Hubs Cloud, and also whether Mozilla has gone through any kind security certifications for Hubs Cloud, such as SOC2 Type 2, CSA-STAR, or HECVAT.   Although AWS has these certifications, we are required to ask vendors about certifications for the actual product, which in this case is Hubs Cloud.  We emailed the Mozilla Hubs support email address (hubs@mozilla.com ) a number of weeks ago and we have had no response.   So we are now trying this method to get in touch with the Mozilla Hubs team.   It would be good for the Mozilla Hubs team to know that more and more universities are requiring the Section 508 compliance documentation (VPAT/ACR), as well as security certifications for cloud platforms and applications.  If there is a specific person on the Hubs team that I can communicate with about the above documentation, can you put me in touch with them and I'll provide them with my work email address for further communication.   Thank you SO much in advance!

**Comment by pattersonbl2** [comment, marked as answer]
We currently don't have this documentation due to legal and different requirements for each state. we will probably work to address these requirements in the future, when we can expand our team.
